packages/autocomplete/src/autocomplete.vue

```diff
@@ -216,8 +228,13 @@
       },
       close(e) {
         this.activated = false;
+        if (!this.selected && !this.blurOnSelect) {
+          this.selected = true;
```

Comment:
Why is `selected` set to true here?

Reply:
I set selected=true so that this won't conditional will not be true again

Reply:
Per the quick chat yesterday, this is really something like "was select handled", so you're saying that the selection was handled at this point (and the field was blurred)? But, I think my confusion comes from assuming that when the autocomplete it closed this prop wouldn't be checked again until it was opened again, and it should go back to its default state (false).